pinot-core/src/main/java/com/linkedin/pinot/core/segment/index/loader/SegmentPreProcessor.java

```diff
@@ -62,6 +63,16 @@ public void process() throws Exception {
       return;
     }
 
+    File[] directoryListing = _indexDir.listFiles();
+    String tempFileExtension = V1Constants.Indexes.BITMAP_INVERTED_INDEX_FILE_EXTENSION + ".tmp";
```

Comment:
I suppose this is coming from OffheapBitmapInvertedIndexCreator, and we are protecting against the server getting killed while the temp files are in place?
Would it make sense to make a call to OffheapBitmapInvertedIndexCreator to clear its tmp files?  Or, would it make sense to simply delete *.tmp (assuming every other class names its droppings with that extension)?